### PR TITLE
Fix PushContainer  handle calls

### PR DESCRIPTION
**Pull request by oacdutra**
**Description**

After receiving the consignment file, the **transferd daemon** calls `handle_container` directly, causing an _InvalidState_ error because the state has _Free_ instead of _AwaitingContainer_ .

To solve that, I changed the PushContainer handle in two steps:

1. If StormApp is **RgbTransfers**, the transferd calls  `handle_container`
2. If StormApp is **FileTransfer**, the transferd calls  `handle_container`

After changes, the daemon started to store the consignment file.

### CODE REVIEW

File: src/stormd/service.rs [hunk 1]
```diff
@@ -197,11 +199,23 @@ impl Runtime {
...
+                                    storm_app: StormApp::FileTransfer,
+                                    container_id: storm::ContainerFullId {
+                                        message_id: MesgId::default(),
+                                        container_id: data.container_id(),
+                                    },
+                                },
+                            }),
+                        ),
+                        _ => (data.container_id(), CtlMsg::ProcessContainer(data)),
+                    },
```
Comment (dr-orlovsky) [inline src/stormd/service.rs]: We can't allow application-specific logic to happen within generic daemon code. There should be another way of fixing that in non-application-specific path.

I am currently sick and my head is not thinking well, so I can't do a suggestion right now - need to do more investigation on the invalid behavior later

File: src/stormd/service.rs [hunk 2]
```diff
@@ -197,11 +199,23 @@ impl Runtime {
 
                 let (container_id, instr) = match mesg {
                     // These should be processed by transfer service
-                    // TODO: Ensure that the incoming chunks references correct app id and message
```
Comment (dr-orlovsky) [inline src/stormd/service.rs]: This todo is not resolved here; its real meaning is that we need to ensure that the app has the access to the chunks, i.e. they were created by the same app originally. Since this PR is not related to that we need to leave TODO here after the bugfix.

**Review by dr-orlovsky — COMMENTED** [review]
Pls see my comments

File: src/stormd/service.rs (hunk 2 above)
Comment (oacdutra) [reply src/stormd/service.rs]: Ok.

File: src/stormd/service.rs (hunk 1 above)
Comment (oacdutra) [reply src/stormd/service.rs]: The change is related with this PR RGB-WG/rgb-node/pull/202. 

I handle the `consignment file` from the `rgb daemon`, I will try extract this logic and create a specific bus message to  **receive by rgb-storm integration** event. Do you agree?

And.. I hope you feel better soon.

**Comment by oacdutra** [comment]
Hi @dr-orlovsky,

I followed your recommendations, removed the application-specific logic, and restored the TODOs.

File: src/stormd/service.rs [hunk 3]
```diff
@@ -155,6 +154,9 @@ impl esb::Handler<ServiceBus> for Runtime {
             (ServiceBus::Rpc, BusMsg::Rpc(msg), ServiceId::Client(client_id)) => {
                 self.handle_rpc(endpoints, client_id, msg)
             }
+            (ServiceBus::Rpc, BusMsg::Storm(msg), other_source) => {
```
Comment (oacdutra) [inline src/stormd/service.rs:157]: @dr-orlovsky , 

This solution works, but I am not sure about the way.

I added this handle because the `bucketd` is marked as `ServiceId::Other`. Is this correct, or is it necessary to add the specific ServiceId to `bucketd`?

**Review by dr-orlovsky — APPROVED** [review]
utACK e60d1c6a23ee7fecd8e7e0c0d1bc17a3563a8277

I think your approach is correct
